### PR TITLE
Overview over different ultrasonic sensors sold as JSN-SR04T

### PR DESCRIPTION
**Pull request by SubOptimal**
Start to collect information about the different ultrasonic boards which might be sold as JSN-SR04T.

the information included for boards:
- AJ-SR04M
- JSN-SR04T
- JSN-SR04T-2.0
- JSN-SR04T-v3.0

### CODE REVIEW

File: content/en/docs/hardware/collective-order.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+documentation:
+
+- https://web.archive.org/web/20180712193526/http://www.ekt2.com/pdf/412_CH_ULTRASONIC_MODULE_4PIN_WATERPROOF.pdf
+- a document found often cited on chinese webpages JSN-SR04T一体化超声波测距说明书.pdf, no free download found 
+
+pictures:
+
+front 1<br>
+![front 1](jsn-sr04t-front-1.png)<br>
```
Comment (opatut) [inline content/en/docs/hardware/collective-order.md:50]: Please use the `imgproc` shortcode, as demonstrated in other files.

**Review by opatut — CHANGES_REQUESTED** [review]
I like the contents of this file and that you started with it *somewhere*, however, your whole formatting is rather careless. This is a public facing website, so please take care to make it look good and appear consistent. If all you want is a place to put the information for later refining it for publishing, create a document on Nextcloud instead, or a post in Slack that you edit, or somthing like that.

If you can't be bothered with formatting, I can do that as well, just tell me. But the way it is now I can't merge it.

File: content/en/docs/hardware/collective-order.md (hunk 1 above)
Comment (SubOptimal) [reply content/en/docs/hardware/collective-order.md:50]: Thanks for your response. I tried yesterday with the `improc` shortcode but did not get it working.

With the files at the same location, I tried the following code:
```
{{< imgproc jsn-sr04t-front-1.png Resize 600x >}}
{{< /imgproc >}}
```

If you could give me a hint on that, I will clean it up and make a new PR:

Comment (SubOptimal) [reply content/en/docs/hardware/collective-order.md:50]: Probably I wis mislead by file `models.md`, which also use the Markdown syntax for file references.

I will push changes later today for review.

**Comment by SubOptimal** [comment]
@opatut Please have a look and let me know if something more needs to be changed.

**Review by opatut — APPROVED** [review]
Thank you :)

**Pull request merged**